client/src/curriculum/topic-icon.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactComponent as ScriptingSVG } from "../../public/assets/curriculum/cur-topic-scripting.svg";
+import { ReactComponent as ToolingSVG } from "../../public/assets/curriculum/cur-topic-tooling.svg";
+import { ReactComponent as StandardsSVG } from "../../public/assets/curriculum/cur-topic-standards.svg";
+import { ReactComponent as StylingSVG } from "../../public/assets/curriculum/cur-topic-styling.svg";
+import { ReactComponent as PracticesSVG } from "../../public/assets/curriculum/cur-topic-practices.svg";
```

Comment:
Could we have an `curriculum/icons.tsx` that re-exports all the SVG across the feature?